File: benchmark.php
<?php

function countFactors($max) {
    $count = 0;
    for ($i = 0; $i <= $max; $i++) {
        if (i % 3 == 0) $count ++;
    }
    return $count;
}

countFactors(10000000); // divide it by 100 because PHP is pretty slow

?>

File: benchmark.c
int countFactors(int max) {
    unsigned long long i, count = 0;
    for (i = 0; i <= max; i++) {
        if (i % 3 == 0) count ++;
    }
    return count;
}

int main() {
    return countFactors(1000000000L);
}
